src/image_copy/main.go

```diff
@@ -119,7 +119,16 @@ func mustHaveArchitecture(imageTag, platform, expectedArch string) {
 		log.Fatal("no binaries were found")
 	}
 	for _, file := range files {
-		out = execCmd("file", filepath.Join(tmpDir, file.Name()))
+		fp := filepath.Join(tmpDir, file.Name())
+		info, err := os.Lstat(fp)
+		if err != nil {
+			log.Fatal(err.Error())
+		}
+		if info.Mode()&os.ModeSymlink != 0 {
+			// skip symlinks
+			continue
+		}
```

Comment:
What kind of symlinks do we have (now)?

Reply:
`cqlsh`: I changed how it's installed, and now it's a symlink.

Reply:
I would recommend doing this instead:
```
If you specify the -L option, docker cp follows any symbolic link in the SRC_PATH.
```
ie add `-L` above instead.

Reply:
I tried, it doesn't really work when the symlink is pointing to a folder outside of the folder you're copying (https://github.com/moby/moby/issues/30213)

Reply:
Ah, too bad!